front_end/src/BacPage.js

```diff
@@ -1,7 +1,5 @@
-
-import React,{useState} from 'react';
-import './bacPage.css';
-
+import React, { useState } from "react";
```

Comment:
I have a linter that automatically formats this. I will highlight the thing I actually did change.